pkg/brokerapi/fake/fake_poller.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fake
+
+import (
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1"
+	"github.com/kubernetes-incubator/service-catalog/pkg/brokerapi"
+)
+
+// Poller implements a fake (./pkg/brokerapi).Poller. Used for unit testing.
+type Poller struct {
+}
+
+// NewPoller creates a poller that responds as specified
+func NewPoller() brokerapi.Poller {
+	return &Poller{}
+}
+
+// CreatePoller starts polling the specified last operation endpoint.
+func (p *Poller) CreatePoller(brokerapi.BrokerClient, *v1alpha1.Instance, *brokerapi.LastOperationRequest, brokerapi.PollerCallback) error {
```

Comment:
Probably add to a queue, point is that in order for a 'follower' to update the condition, these are necessary to be kept for a context.